packages/salesforcedx-vscode-core/test/vscode-integration/commands/functions/forceFunctionContainerStartCommand.test.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { vscodeStub } from '@salesforce/salesforcedx-utils-vscode/out/test/unit/commands/mocks';
+import * as proxyquire from 'proxyquire';
+import { assert, createSandbox, SinonSandbox, SinonStub, stub } from 'sinon';
+
+const proxyquireStrict = proxyquire.noCallThru();
+
+describe.only('Force Function Start Container Command Unit Tests.', () => {
```

Comment:
Maybe I'm not seeing something here, but both forceFunctionContainerStartCommand.test.ts and forceFunctionContainerStart.test.ts test forceFunctionContainerStartCommand.  Can these be consolidated into just forceFunctionContainerStartCommand.test.ts?  We like to have a 1:1 mapping between test and implementation files.

Reply:
So this file is the unit tests and the `forceFunctionContainerStart.test.ts` is the integration tests.  I updated the descriptions in the describes for each file, but didn't want to move things around just yet until we dig deeper on what we're going to do with the unit tests for this package.